services/QuillLMS/spec/controllers/auth/classlink_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Auth::ClasslinkController, type: :controller do
+  describe '#classlink' do
```

Comment:
You could refactor: `subject { get :classlink, params: { code: }` at the top here.  That way you can quickly scan that all tests are hitting the same endpoint.